### ISSUE TITLE
[REQ] Recomended Typescript Generator?

### ISSUE THREAD

**Issue by ihqtim** [OpenAPITools/openapi-generator#16899]
### Is your feature request related to a problem? Please describe.

Thanks for this awesome project. About a year ago started using openapi-generator to create typescript clients and selected the generator "[typescript (experimental)](https://openapi-generator.tech/docs/generators/typescript)" - primarily as it seemed to work well for browser & node. It has worked so far, no complaints.

Performing a review and there remain multiple typescript generators; wondering how to gauge which one/ones are most actively maintained and used.

## Describe the solution you'd like

Looking for recommendations regarding the most popular, well maintained, typescript client generator that can emit a single typescript client suited for use in both browser and modern nodejs (which include fetch) applications. Thanks!

## Describe alternatives you've considered

Have scoured the web and the issues register looking for recommendations.

**Comment by YoavBZ** [comment]
@ihqtim Did you find the info you were looking for? I'm also interested which generator should be used

**Comment by ihqtim** [comment]
> Did you find the info you were looking for

Nope, didn't get any responses, I remain unsure which one is recommended by the team. In the absence of guidance I went with "[typescript (experimental)](https://openapi-generator.tech/docs/generators/typescript)" and it is working OK for my use case.

**Comment by bodograumann** [comment]
Are there any plans of making the experimental typescript generator the default and deprecate the others, @TiFu (2017/07) @taxpon (2017/07) @sebastianhaas (2017/07) @kenisteward (2017/07) @Vrolijkx (2017/09) @macjohnny (2018/01) @topce (2018/10) @akehir (2019/07) @petejohansonxo (2019/11) @amakhrov (2020/02) @davidgamero (2022/03) @mkusaka (2022/04) @joscha (2024/10)?
